content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,174 @@
+the checkbox next to :guilabel:`FedEx Connector`.
+
+Finally, click :guilabel:`Save` to save the changes. After doing so, a :icon:`oi-arrow-right`
+:guilabel:`FedEx Shipping Methods` button appears below :guilabel:`FedEx Connector`.
+
+.. image:: fedex/fsm-button.png
+   :align: center
+   :alt: The FedEx Shipping Methods button below the FedEx Connector.
+
+Configure shipping method
+=========================
+
+Once the FedEx shipping connector is enabled, it is necessary to configure at least one shipping
+method. After doing so, the shipping method can be included in sales orders (SOs), and used to
+compute shipping costs, and print shipping labels.
+
+To enable a shipping method, navigate to :guilabel:`Inventory app --> Configuration --> Settings`,
+and click the :guilabel:`FedEx Shipping Methods` button below the :guilabel:`FedEx Connector`
+checkbox. Doing so opens a page that shows all existing FedEx shipping methods.
+
+.. note::
+   To see all shipping methods for every shipper with a connector enabled, navigate to
+   :menuselection:`Inventory app --> Configuration --> Shipping Methods`.
+
+Select a shipping method to open its form. Alternatively, click :guilabel:`New` to open a blank
+form, and configure a new shipping method.
+
+.. image:: fedex/fedex-form.png
+   :align: center
+   :alt: The form for a FedEx shipping method.
+
+.. important::
+   Enabling the FedEx shipping connector automatically creates two default shipping methods:
+   :guilabel:`FedEx US` and :guilabel:`FedEx International`. Each of these methods are
+   pre-configured with test credentials, allowing them to be used for testing purposes.
+
+   Before the shipping method can be used to create actual shipments, the test credentials must be
+   replaced with credentials from a valid FedEx account.
+
+General information
+-------------------
+
+At the very top of a shipping method form are fields used to configure the way the method operates
+in Odoo. In the :guilabel:`Provider` field, select :guilabel:`FedEx` from the drop-down menu, if it
+is not already selected.
+
+The rest of the fields in this section are general to all shipping providers. For details on how to
+fill them out, see the documentation on :doc:`third-party shippers <third_party_shipper>`.
+
+Fedex Configuration tab
+-----------------------
+
+The options in the :guilabel:`Fedex Configuration` tab of a FedEx shipping method form are used to
+connect the method to a FedEx account, and configure the shipping details associated with the method
+(drop-off type, package type, etc.).
+
+A FedEx business account is required to obtain the information needed to fill out the fields in this
+tab. To create a new account, navigate to FedEx's `Open Account
+<https://www.fedex.com/en-us/open-account.html>` page, click on :guilabel:`Create Account`, and
+follow the instructions.
+
+Developer Key and Meter Number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Similar to adding "tab" on the _Fedex Configuration_ heading, I suggest adding "fields" to this heading to help define that these are fields within the form and not a section or tab
```suggestion
Developer Key and Meter Number fields
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```